codebase/src/sorting/components/ArrayInput.vue

```diff
@@ -1,258 +1,283 @@
+  }
+  for (const num of numbers) {
+    if (num > 50 || num < 0) {
+      error.value = "Values should be between 0 and 50";
+      return;
     }
-    for (const num of numbers) {
-        if (num > 50 || num < 0) {
-            error.value = 'Values should be between 0 and 50';
-            return;
-        }
-        if (isNaN(num)) {
-            error.value = 'Invalid number detected';
-            return;
-        }
+    if (isNaN(num)) {
+      error.value = "Invalid number detected";
+      return;
     }
-    array.value = numbers;
-    error.value = '';
+  }
+  array.value = numbers;
+  error.value = "";
 };
 
 const clearArray = () => {
-    array.value = [];
-    inputValue.value = '';
-    emit('sortedArray', array.value);
-    resetValue();
+  array.value = [];
+  inputValue.value = "";
+  emit("sortedArray", array.value);
+  resetValue();
 };
 
 const sortArray = async () => {
-    await bubbleSort(array.value, 500, updateSwap, updateLeftIndex, updateRightIndex, updateCurrentLines, updateSortedIndex);
-    emit('sortedArray', array.value);
-    resetValue();
+  await bubbleSort({
+    array: array.value,
+    ms: 500,
+    updateSwap: updateSwap,
+    updateLeftIndex: updateLeftIndex,
+    updateRightIndex: updateRightIndex,
+    updateCurrentLines: updateCurrentLines,
+    updateSortedIndex: updateSortedIndex,
+  });
+  // await insertionSort({
+  //     array: array.value,
+  //     ms: 500,
+  //     updateSwap: updateSwap,
+  //     updateLeftIndex: updateLeftIndex,
+  //     updateRightIndex: updateRightIndex,
+  //     updateCurrentLines: updateCurrentLines,
+  // });
+  emit("sortedArray", array.value);
+  resetValue();
 };
 
 const generateArray = () => {
-    const randomArray = Array.from({ length: 10 }, () => Math.floor(Math.random() * 50));
-    array.value = randomArray;
+  const randomArray = Array.from({ length: 10 }, () =>
+    Math.floor(Math.random() * 50)
+  );
+  array.value = randomArray;
 };
 
 const updateSwap = (newArray: number[]) => {
-    array.value = newArray;
+  array.value = newArray;
 };
 </script>
 
 <style scoped>
 .arrayInput {
-    display: flex;
-    flex-direction: column;
-    height: 100vh;
-    padding: 20px;
-    box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  height: 100vh;
+  padding: 20px;
+  box-sizing: border-box;
 }
 
 .form-container {
-    display: flex;
-    flex-direction: column;
-    justify-content: center;
-    align-items: center;
-    background-color: #f7f9fc;
-    padding: 20px;
-    position: absolute;
-    bottom: 20px;
-    left: 20px;
-    border-radius: 8px;
-    width: auto;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: #f7f9fc;
+  padding: 20px;
+  position: absolute;
+  bottom: 20px;
+  left: 20px;
+  border-radius: 8px;
+  width: auto;
 }
 
 input[type="text"] {
-    font-size: 14px;
-    width: auto;
+  font-size: 14px;
+  width: auto;
 }
 
 .form-styles {
```

Comment:
these css class names are not very descriptive, and putting a ton of them together like this makes the styles much harder to debug

Reply:
I improved some of them